plugins/modules/ansible_sync.py

```diff
@@ -35,6 +35,10 @@
       - Name of the repository
     type: str
     required: true
+  mirror:
+    description:
+      - If True, synchronization will run in "mirror" mode instead of "additive"
```

Comment:
Maybe explain that mirror mode removes content that disappeared upstream.
Also False means additive (no content is removed).
If not specified the server default is used.
(see my other comment for this tri-state)